conda_recipes/pyelasticsearch/meta.yaml

```diff
@@ -0,0 +1,74 @@
+package:
+  name: pyelasticsearch
+  version: "1.4"
+
+source:
+  fn: pyelasticsearch-1.4.tar.gz
+  url: https://pypi.python.org/packages/source/p/pyelasticsearch/pyelasticsearch-1.4.tar.gz
+  md5: ed61ebb7b253364e55b4923d11e17049
+#  patches:
+   # List any patch files here
+   # - fix.patch
+
+build:
+  noarch_python: True
+  # preserve_egg_dir: True
+  # entry_points:
+    # Put any entry points (scripts to be generated automatically) here. The
+    # syntax is module:function.  For example
+    #
+    # - pyelasticsearch = pyelasticsearch:main
+    #
+    # Would create an entry point called pyelasticsearch that calls pyelasticsearch.main()
+
+
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  # number: 1
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - certifi
+    - elasticsearch-py >=1.3.0,<2.0.0
```

Comment:
Note that the PyPI name is "elasticsearch", but this conflicts with the actual elasticsearch package.